plugins/omprog/omprog.c

```diff
@@ -398,6 +394,14 @@ cleanup(wrkrInstanceData_t *pWrkrData)
 	RETiRet;
 }
 
+BEGINfreeWrkrInstance
+CODESTARTfreeWrkrInstance
+	if (pWrkrData->bIsRunning) {
+		kill(pWrkrData->pid, SIGTERM);
```

Comment:
Sure, let us call it signalOnClose (signalTERMOnClose seems too verbose). 

Just to re-iterate, it'll default to off (existing behavior), if turned-on the new behavior will issue a TERM and reap the process.

I'll start working on it in a day or two.